intact-orthology-import/src/main/java/uk/ac/ebi/intact/ortholog/OrthologsFileParser.java

```diff
@@ -44,4 +46,45 @@ public static Map<String, String> parseFile(String filePath) {
         log.info("Number of Panther identifiers: " + uniprotAndPTHR.size());
         return uniprotAndPTHR;
     }
+
+    public static void parseFileAndSave(String inputFilePath, String outputFilePath) throws IOException {
```

Comment:
Instead of parsing the orthologs file and creating a map, we instead save the pairs to the file, having the equivalent of the map in a file.